packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.0.3 - Jul 25, 2022
+
+Incremental version for Teams Toolkit with multiple bugs fix and enhancements.
+
+Enhancement:
```

Comment:
We also have SPFx upgraded to v1.15.0 and would like to notify users about that. cc @xu-hong

Reply:
thank you, added.